components/ai_chat/content/browser/page_content_fetcher.cc

```diff
@@ -227,8 +253,62 @@ class PageContentFetcher {
   base::WeakPtrFactory<PageContentFetcher> weak_ptr_factory_{this};
 };
 
+#if BUILDFLAG(ENABLE_TEXT_RECOGNITION)
+void OnGetTextFromImage(
+    base::OnceCallback<void(std::string, bool is_video)> callback,
+    const std::pair<bool, std::vector<std::string>>& supported_strs) {
```

Comment:
To remove it would mean you can't tell the difference between a failure and no text in a screenshot. I think it should be kept because despite the handling is the same in this code and in the dialog view, knowing when there is an error is generally good. Thoughts?

Reply:
I'm fine keeping the change but I think it should be propagated through the callback otherwise we still can't tell between a failure and no text. Or at least we do an early return when it fails without going through the vector.

Reply:
Sounds good, updated in 1b2d87ff5358740b03e84da39f396084dd0e091f and also added a test for unsupported locale for Windows.
I combined the 4 browsertests into 1 because it was causing a timeout with 4.